src/sys/context.rs

```diff
@@ -0,0 +1,25 @@
+use libc;
+use {Errno, Result};
+use std::mem;
+
+#[derive(Clone, Copy)]
+pub struct UContext {
+    context: libc::ucontext_t,
```

Comment:
I would not mind. Feel free to add a corresponding section to the CONVENTIONS, as you have already talked about in the other issue. We should also open a issue that tracks the refactoring process.

Unless we have public members in our structs anywhere, it will not even be a breaking change.

I will leave this unchanged until we have that convention merged into master.